lib/squadster/helpers/permissions.ex

```diff
@@ -32,6 +37,17 @@ defmodule Squadster.Helpers.Permissions do
     user |> can_update?(squad_member)
   end
 
+  defp user_squad(user) do
+    ((user
```

Comment:
Interesting style, but in this case it will be better to do
```elixir
user |> Repo.preload(squad_member: %{squad: :members})
```

Reply:
:+1:

Reply:
Honestly, this request should look like `user |> Repo.preload(squad_member: [:squad, squad: :members])`